### ISSUE TITLE
What has changed after #111

### ISSUE THREAD

**Issue by ostcar** [OpenSlides/openslides-permission-service#114]
Please tell me what has changed after #111

**Comment by ostcar** [comment]
@normanjaeckel Can this be closed?

**Comment by normanjaeckel** [comment]
No, all write perms are still open.

**Comment by normanjaeckel** [comment]
https://github.com/OpenSlides/openslides-permission-service/blob/55d5c676a7707c25d52198b3492da011113d604f/internal/collection/autogen.go#L89

https://github.com/OpenSlides/openslides-permission-service/blob/55d5c676a7707c25d52198b3492da011113d604f/internal/collection/autogen.go#L90

Here you have two small errors. `motion` instead of `motions`

**Comment by normanjaeckel** [comment]
https://github.com/OpenSlides/openslides-permission-service/blob/55d5c676a7707c25d52198b3492da011113d604f/internal/collection/autogen.go#L103

Here another typo

**Comment by normanjaeckel** [comment]
https://github.com/OpenSlides/openslides-permission-service/blob/55d5c676a7707c25d52198b3492da011113d604f/internal/collection/autogen.go#L137

You can not change your own password. So this might be a special case.

**Comment by normanjaeckel** [comment]
https://github.com/OpenSlides/openslides-permission-service/blob/55d5c676a7707c25d52198b3492da011113d604f/internal/collection/autogen.go#L94

This is also a special case. Also for motion.update: Only users with "motion.can_support" can make themnselves as supporters.

**Comment by ostcar** [comment]
I fixed the typos locally. Everything else has its own issue. So after the typo-fix is merged, this will be closed.
